### ISSUE TITLE
TU to existing LU: selectable HUs include others than existing LUs

### ISSUE THREAD

**Issue by metas-dh** [metasfresh/metasfresh-webui-api#783]
### Is this a bug or feature request?
Bug
### What is the current behavior?
When you try to put a TU on an existing LU in Handling Unit Editor window, you can select HUs as LU which are others than the existing LUs.
#### Which are the steps to reproduce?
1. Open HU Editor window (have sev. HUs in it, LUs, TUs, etc.)
2. Select a HU, call Acion Transform
3. Select "TU to existing LU"
4. Check field for selecting LU: not only the existing LUs are selectable, but other HUs as well:

![2018-01-16 10_06_19-metasfresh](https://user-images.githubusercontent.com/18696715/34987886-07daa8b0-fabd-11e7-869f-e77ac5ef4de2.png)

>  "known" issue. 
Long story short, the explanation is: in case of Handling Unit editor window, atm we cannot afford to filter the LUs in memory (the way how is impl. now).
(technically, de.metas.ui.web.handlingunits.process.WebuiHUTransformParametersFiller.getLUsLookupValues(LookupDataSourceContext)).

### What is the expected or desired behavior?
Only the existing LUs shall be selectable.

**Comment by teosarca** [comment]
@metas-dh  @metas-lc isn't this issue similar to https://github.com/metasfresh/metasfresh-webui-api/issues/753 ?

**Comment by metas-lc** [comment]
@teosarca yes. Looks like the same bug
